palak/src/pages/Front.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+// import Button from "../components/Button";
+import LoaderButton from "../components/Button";
+import { useNavigate } from "react-router-dom";
+
+function Front() {
+  const navigate = useNavigate();
+  const handleLogout = () => {
+    localStorage.removeItem("jwtToken");
+    navigate("/");
+  };
+  return (
+    <div className="flex items-center justify-center flex-col">
+      <h1 className="text-3xl mt-10">Dashboard</h1>
```

Comment:
Here since you have already defined the text size, then there's no point of using h1 tag. Instead you can use p tag as h1 properties here are overwritten by the css.